### PR TITLE
#1288 - Update of sidebar links

### PR DESCRIPTION
**Pull request by Jeckerson**
Hello!

* Type: documentation
* Link to issue: #1288 

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines][:contrib:]
- [x] I have checked that another pull request for this purpose does not exist
- [ ] I wrote some tests for this PR

Small description of change:

Screenshot from 2019-03-22 09-41-59

Thanks

[:contrib:]: https://github.com/phalcon/phalcon-devtools/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: resources/elements/sidebar-menu.php [hunk 1]
```diff
@@ -192,7 +192,7 @@
         // Phalcon Docs
         [
             'link'  => [
-                'href'  => 'https://docs.phalconphp.com/en/3.2',
+                'href'  => 'https://docs.phalconphp.com/3.4/en/',
```
Comment (sergeyklay) [inline resources/elements/sidebar-menu.php]: https://phalcon.link/docs

File: resources/elements/sidebar-menu.php [hunk 2]
```diff
@@ -224,5 +224,29 @@
                 'wrap'  => 'span',
             ],
         ],
+
+        // Phalcon Official Forum
+        [
+            'link' => [
+                'href' => 'https://forum.phalconphp.com/',
```
Comment (sergeyklay) [inline resources/elements/sidebar-menu.php]: https://phalcon.link/forum

Comment (Jeckerson) [reply resources/elements/sidebar-menu.php]: 8fe8dfe

File: resources/elements/sidebar-menu.php (hunk 1 above)
Comment (sergeyklay) [reply resources/elements/sidebar-menu.php]: Could we change this as well

Comment (Jeckerson) [reply resources/elements/sidebar-menu.php]: c90405b

I got some doubts about transition time from 3.4 to 4.0 and have dynamic link already pointed to 4.0 version when we still could have 3.4 in active usage. But after some thinking - better to have actual link to fresh version...

**Pull request merged**